west.yml

```diff
@@ -129,7 +129,7 @@ manifest:
           compare-by-default: true
     - name: mcuboot
       repo-path: sdk-mcuboot
-      revision: beebb23f8e2d744ef89819e584928377ef6c444a
+      revision: pull/294/head
```

Comment:
You should use the commit SHA here, since the PR was already merged: `b3c97b010983e5523d90f9140b8e657b06a2ea3f`